DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py

```diff
@@ -62,14 +62,17 @@
     ## [optional] : when omitted all monitoring plots for triggering
     ## will be empty
     triggerExtras = cms.PSet(
-        src = cms.InputTag("TriggerResults","","HLT"),
-        pathsELECMU = cms.vstring([ 'HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v', 
-                                    'HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v']),
-        pathsDIMUON = cms.vstring([ 'HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v',
-                                    'HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_DZ_v']),
-        pathsDIELEC = cms.vstring([ 'HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v', 
-                                    'HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_v'])
-    )    
+		src = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
Please don't use tabs in python. Especially not if the rest of the file uses spaces.